File: BackEnd_ExpressJS/app/routes/authRoutes.js
const apiController = require('../controllers/apiController');
const express = require('express');
const router = express.Router();
router.use(express.json());

const { protectRoute } = require('../config/auth');

router.get('/', protectRoute, apiController.index);
router.get('/tests', protectRoute, apiController.tests);
router.get('/user-info', protectRoute, apiController.userInfo);
router.get('/list-menu', protectRoute, apiController.listMenus);

module.exports = router;

File: BackEnd_Django/api/myRoutes/authRoutes.py
from rest_framework import routers
from django.urls import path, include
from ..myViewers import viewTranslations, viewApi
from django.views.decorators.csrf import csrf_exempt

myRouters = routers.DefaultRouter()
myRouters.register(r'translations', viewTranslations.ViewSet)

urlpatterns = [
    path('', include(myRouters.urls)),
    path('', csrf_exempt(viewApi.index), name='api-index'),
    path('tests', csrf_exempt(viewApi.tests), name='tests'),
    path('user-info', csrf_exempt(viewApi.userInfo), name='user-info'),
    path('list-menu', csrf_exempt(viewApi.listMenus), name='list-menu'),
]